app-admin/kubelet/files/kubelet

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Wrapper for launching kubelet via rkt-fly stage1
+
+set -e
+
+if [ -z "${KUBELET_VERSION}" ]; then
+    echo "ERROR: must set KUBELET_VERSION"
+    exit 1
+fi
+
+KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+
+mkdir --parents /etc/kubernetes
+mkdir --parents /var/lib/docker
+mkdir --parents /var/lib/kubelet
+mkdir --parents /run/kubelet
+
+exec /usr/bin/rkt run \
+  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
+  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
+  --volume var-lib-docker,kind=host,source=/var/lib/docker \
+  --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+  --volume run,kind=host,source=/run \
+  --mount volume=etc-kubernetes,target=/etc/kubernetes \
+  --mount volume=etc-ssl-certs,target=/etc/ssl/certs \
+  --mount volume=var-lib-docker,target=/var/lib/docker \
+  --mount volume=var-lib-kubelet,target=/var/lib/kubelet \
+  --mount volume=run,target=/run \
+  $RKT_OPTS \
+  --stage1-image=/usr/share/rkt/stage1-fly.aci \
+  ${KUBELET_ACI}:${KUBELET_VERSION} --exec=/hyperkube -- kubelet "$@"
```

Comment:
Just to be clear, there are two reasons we need to use a kublet aci instead:

 - kublet is a smaller compatibility requirement, it is less clear how hyperkube will evolve over time.
 - the calling convention of the aci must be preserved over time, doing --exec=/hyperkube commits us to always doing hyperkube, a kublet-only aci cannot be substituted. And hyperkube doesn't support aliasing via symlinks so it isn't possible to have --exec=/kublet where kublet is either a real kublet or a symlink to hyperkube. If hyperkube eventually gains the ability to respect argv[0] then it is possible for a user to specify a hyperkube aci as the kublet aci if that is useful for caching reasons or whatnot.